CHANGELOG.md

```diff
@@ -10,6 +10,17 @@
 > - :house: [Internal]
 > - :nail_care: [Polish]
 
+# 11.0.0-alpha.6 (Unreleased)
+
+#### :boom: Breaking Change
+
+- `-bs-super-errors` flag has been removed along with Super_errors. https://github.com/rescript-lang/rescript-compiler/pull/6199
```

Comment:
I cut the alpha.5 release before the super errors changes, so those need to go to alpha.6.